# MAG

Header header
float32 MOfsX
float32 MOfsY
float32 MOfsZ
float32 MagX
float32 MagY
float32 MagZ
float32 OfsX
float32 OfsY
float32 OfsZ
